src/platform_impl/linux/x11/util/cursor.rs

```diff
@@ -170,32 +170,45 @@ pub enum SelectedCursor {
     Named(CursorIcon),
 }
 
+impl Default for SelectedCursor {
```

Comment:
Nit: Use `derive(Default)` + `#[default]` on the enum case.

Reply:
Can not use on algebraic types.